packages/react-components/src/molecules/CalendarLink.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect, useRef } from 'react';
+import css from '@emotion/css';
+import { Button, Anchor } from '../atoms';
+import { perRem } from '../pixels';
+import {
+  chevronUpIcon,
+  chevronDownIcon,
+  linkIcon,
+  outlookIcon,
+  googleCalendarIcon,
+} from '../icons';
+import {
+  paper,
+  steel,
+  colorWithTransparency,
+  tin,
+  mint,
+  lead,
+  pine,
+} from '../colors';
+import { iCalIconImage } from '../images';
+
+const containerStyles = css({
+  alignItems: 'flex-end',
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const menuWrapperStyles = css({
+  position: 'relative',
+});
+
+const menuContainerStyles = css({
+  position: 'absolute',
+  display: 'none',
+  top: `${-6 / perRem}em`,
+  right: `${-6 / perRem}em`,
+
+  backgroundColor: paper.rgb,
+  border: `1px solid ${steel.rgb}`,
+  boxShadow: `0 2px 6px 0 ${colorWithTransparency(tin, 0.34).rgba}`,
+
+  flexDirection: 'column',
+
+  boxSizing: 'border-box',
+  padding: `${6 / perRem}em 0`,
+});
+
+const showMenuStyles = css({
+  display: 'flex',
+});
+
+const calendarLinkListStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  listStyle: 'none',
+  margin: 0,
+  padding: 0,
+
+  '& li': {
```

Comment:
```suggestion
  '& > li': {
```
just to be safe, who knows what else we'll put somewhere deep down in there